README.md

```diff
@@ -21,5 +21,6 @@ This project uses the [Contributor Covenant](https://github.com/saikobee/squiggl
 - Click the "Fork" repository button and locally clone your fork
 - Use a terminal and `cd` into the new directory
 - Install [Node.js](https://nodejs.org/en/)
+- npm install (this will install the dependencies needed)
```

Comment:
Would you mind wrapping the `npm install` in backticks so it shows up like code?